### DISCUSSION TITLE
User connection logs

### DISCUSSION THREAD

**Discussion by speedlnx** [DefGuard/defguard#868] [General]
Hello, is there any audit log or a way to log and view users login and connection?

**Comment by kchudy** [comment]
Hi @speedlnx , you can find the events in our [Activity log feature](https://docs.defguard.net/features/activity-log). Here's a list of all [available events](https://docs.defguard.net/features/activity-log#events-tracked-in-activity-log). The ones you're interested in are: UserLogin and ConnectedToLocation/ConnectedToMfaLocation.
